lib/seek/samples/extractor.rb

```diff
@@ -38,6 +39,9 @@ def persist
             samples = Sample.where(sample_type: sample_type, title: samples.collect(&:title), contributor: contributor).where(
               'id > ?', last_id
             )
+            samples.each do |sample|
+              sample.project_ids = project_ids
+            end
```

Comment:
I'm not sure this is needed, if the project_ids for the first sample were correct, and the callbacks have been run with run_callbacks

It would be good to either add or expand a test to check `SetSubscriptionsForItemJob` are queued correctly